lib/openid_connect/authorization/endpoint_start_point.rb

```diff
@@ -0,0 +1,34 @@
+module OpenidConnect
+  module Authorization
+    class EndpointStartPoint < Endpoint
+      def initialize(current_user)
+        super(current_user)
+      end
```

Comment:
If all you do is calling the parent with the same parameters, just skip the definition.